src/components/Footer/Footer.css

```diff
@@ -0,0 +1,20 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:400,500,600,700,800,900|Roboto&display=swap');
+
```

Comment:
Use custom properties (CSS variables) for properties that change often. e.g:

```css
:root {
  --font: 'Poppins', sans-serif;
  font-family: var(--font);
  font-style: normal;
}

.someSelectorWithRobotoText {
  --font: Roboto;
}
```

There's a base font-size already set by tailwind, use `rem` or `em` values in your code base for font-sizes and pixel widths. e.g

```css
.selector {
  padding: 4px;
  /* becomes */
   padding: 0.25rem; 
   /* assuming 1rem = 16px */
}
```